src/Ocelot.Provider.Kubernetes/KubeServiceCreator.cs

```diff
@@ -34,6 +34,8 @@ public virtual IEnumerable<Service> CreateInstance(KubeRegistryConfiguration con
         return new Service[] { instance };
     }
 
+    protected IOcelotLogger Logger => _logger;
```

Comment:
Why not use auto-property instead?

Reply:
Fixed, see https://github.com/ThreeMammals/Ocelot/pull/2111/commits/c4c7ae0ff2a891d64b4f81b6a6d051cc15e71211#diff-aa904908dbabbf75561309a7e6be8cca8c423ac49d0c1d7a77bd8dd609316b17